### PR TITLE
Port graphics trends from databricks to dataproc

### PR DESCRIPTION
**Pull request by BenWu**
Migration of graphics-telemetry-dashboard-big-query and graphics-telemetry-trends-big-query jobs from databricks.  Intention is to use this repo as storage for the python driver code for a dataproc pyspark job.  Associated airflow pr https://github.com/mozilla/telemetry-airflow/pull/1191

Code is mostly unchanged; this is using boto3 for s3 access and adds cli arguments

The notebooks:
https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/239724
https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/240233

circleci deploy keys for this repo were deleted so ci doesn't work but I don't think it's needed.

### CODE REVIEW

File: mozetl/graphics/create_cluster.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (akkomar) [inline mozetl/graphics/create_cluster.sh:1]: Are these scripts for testing? It's not clear, some comment about this would be nice.

**Review by akkomar — CHANGES_REQUESTED** [review]
Notebook code looks good, let's not dive deep into that :)

One nit about scripts - I guess they're for testing purposes?

> circleci deploy keys for this repo were deleted so ci doesn't work but I don't think it's needed.

We should fix that before merging (linting can fail here).

File: mozetl/graphics/create_cluster.sh (hunk 1 above)
Comment (BenWu) [reply mozetl/graphics/create_cluster.sh:1]: Scripts are for testing, I'll add a comment.

**Comment by BenWu** [comment]
I formatted code and updated dependencies in setup.py.  Still some failing ci for other reasons. I'm at eod and can't look closer but I'll revisit tomorrow.

**Review by akkomar — APPROVED** [review]
It's not immediately clear to me what's wrong with the tests.
I'm giving r+ because I think we're under time pressure here.
Later on we should figure out what's wrong with the build.

**Comment by BenWu** [comment]
I created issue https://github.com/mozilla/python_mozetl/issues/352 listing the CI errors.  I'll take another look later

**Pull request merged**